src/SvgParser.elm

```diff
@@ -285,18 +312,23 @@ parseToNode input =
 parse : String -> Result String (Html msg)
 parse input =
     let
-        toHtml svgNode =
-            case svgNode of
-                SvgElement element ->
-                    if element.name == "svg" then
-                        Ok <|
-                            svg (List.map toAttribute element.attributes)
-                                (List.map nodeToSvg element.children)
-
-                    else
-                        Err "Top element is not svg"
-
-                _ ->
+        toHtml svgNodes =
+            case svgNodes of
+                svgNode :: tl ->
+                    case svgNode of
+                        SvgElement element ->
+                            if element.name == "svg" then
+                                Ok <|
+                                    svg (List.map toAttribute element.attributes)
+                                        (List.map nodeToSvg element.children)
+
+                            else
+                                toHtml tl
+
+                        _ ->
+                            toHtml tl
+
+                [] ->
```

Comment:
The error message is no longer correct. The top element could never be a svg if there is no top element. Write something like `"No svg found"`.